flexmeasures/app.py

```diff
@@ -150,6 +155,27 @@ def create(  # noqa C901
     def before_request():
         if app.config.get("FLEXMEASURES_PROFILE_REQUESTS", False):
             g.start = time.time()
+        if app.config.get("FLEXMEASURES_PROFILE_PYINSTRUMENT", False):
```

Comment:
In the other code, we rule out certain requests (namely the favicon). Seems appropriate in general for profiling, I believe.